common/theme.c

```diff
@@ -14,6 +14,20 @@ bool colorFromSetting(config_setting_t *rgba, color_t *col) {
 }
 
 void themeStartup(ThemePreset preset) {
+    globalPreset = preset;
+    char* buttonAText = calloc(7,sizeof(char));/*Must initialize the character arrays on the heap*/
+    char* buttonBText = calloc(7,sizeof(char));/*so that they are not destroyed whenever themeStartup function ends*/
+    switch (preset) {
+        case THEME_PRESET_LIGHT:
+        default:
+            strcpy(buttonAText, "\uE0E0");
+            strcpy(buttonBText, "\uE0E1");
+            break;
+        case THEME_PRESET_DARK:
+            strcpy(buttonAText, "\uE0A0");
+            strcpy(buttonBText, "\uE0A1");
+            break;
+    }
     theme_t themeLight = (theme_t) { 
```

Comment:
This should be static const.

Reply:
so ininitalizing it as a static const will keep it from being destroyed whenever the function goes out of scope? does this mean i can remove the calloc and go back too the literals?

Reply:
uh, ive google the hell outta this and cannot find a way to make this a static const in C

Reply:
i tried 
static const theme_t themeLight = (const theme_t) {/*data*/}; //cant do this because idk why
static const theme_t themeLight = { /* data*/}; //cant do this because MakeColor() isnt a const
static const theme_t *themeLight = ((&const theme_t) {/*data*/});//cant do this because idk why

Reply:
the solution yellows8 provided below was able to resolve this and get rid of the need for calloc